zwave_js_server/model/node/__init__.py

```diff
@@ -644,6 +644,14 @@ async def async_set_location(
         )
         self.data["location"] = location
 
+    async def async_get_firmware_update_progress(self) -> bool:
+        """Send getFirmwareUpdateProgress command to Node."""
+        data = await self.async_send_command(
+            "get_firmware_update_progress", require_schema=18, wait_for_result=True
+        )
+        assert data
+        return cast(bool, data["progress"])
```

Comment:
Does a `True` return value mean that a firmware update is in progress, and `False` means there's no firmware update in progress?

Reply:
That's correct. Should I add that to the docstring?

Reply:
Yes, I think it's needed.